modules/photo/src/seamless_cloning_impl.cpp

```diff
@@ -329,7 +329,7 @@ void Cloning::normalClone(const Mat &destination, const Mat &patch, Mat &binaryM
 
     computeDerivatives(destination,patch,binaryMask);
 
-    switch(flag)
```

Comment:
The condition is a bit hacky and depends on enum values. I propose to extend "case:" sentences and/or add explicit conditions to make _ALT branch obvious from the code for reader.